src/challenges/challenge-cache.ts

```diff
@@ -0,0 +1,373 @@
+// const dbc = require('../dbc.js')
+// const config = require('../../config.json')
+
+import ChainWar from './models/chainwar'
+import ChallengeStates from './models/challenge-states'
+import ChallengeTypes from './models/challenge-types'
+import Sprint from './models/sprint'
+import War from './models/war'
+import { Client, Collection, GuildChannel, Message, Snowflake, User } from 'discord.js'
+
+declare const client: Client
+
+enum FieldOptions {
+  X_STATUS = 'Cross-Server Display',
+  AUTO_STATUS = 'Automatic Summaries',
+  PREFIX = 'Prefix',
+  ANNOUNCE = 'Announcements Channel',
```

Comment:
Only the first two of these are speficially relevant to challenges?  Were they handled here before?

Reply:
https://github.com/aigeroni/Winnie_Bot/blob/main/src/challenges/clist.js#L156

Reply:
Look who doesn't remember their own code!  😖